bufr_ncep_prepbufr_adpupa_3test.yaml

```diff
@@ -0,0 +1,264 @@
+          timeCycleProfileLevel:
+            query: "*/PRSLEVEL/DRFTINFO/HRDR"
+            
+          # ObsValue
+          dewpointTemperature:
+            query: "*/PRSLEVEL/Q___INFO/TDO"
+            transforms:
+              - offset: 273.15
+          virtualTemperature:
+            query: "*/PRSLEVEL/T___INFO/TVO"
+            transforms:
+              - offset: 273.15
+          pressure:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT/POB"
+            transforms:
+              - scale: 100
+          specificHumidity:
+            query: "*/PRSLEVEL/Q___INFO/Q__EVENT/QOB"
+            transforms:
+              - scale: 0.000001
+          airTemperature:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT/TOB"
+            transforms:
+              - offset: 273.15
+          heightOfObservation:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT/ZOB"
+          windEastward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/UOB"
+          windNorthward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/VOB"
+                        
+          # QualityMarker
+          pressureQualityMarker:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT/PQM"
+          specificHumidityQualityMarker:
+            query: "*/PRSLEVEL/Q___INFO/Q__EVENT/QQM"
+          airTemperatureQualityMarker:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT/TQM"
+          heightQualityMarker:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT/ZQM"
+          windQualityMarker:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/WQM" 
+
+          # ObsError
+          pressureError:
+            query: "*/PRSLEVEL/P___INFO/P__BACKG/POE"
+            transforms:
+              - scale: 100
+          relativeHumidityError:
+            query: "*/PRSLEVEL/Q___INFO/Q__BACKG/QOE"
+          airTemperatureError:
+            query: "*/PRSLEVEL/T___INFO/T__BACKG/TOE"
+            transforms:
+              - offset: 273.15
+          windError:
+            query: "*/PRSLEVEL/W___INFO/W__BACKG/WOE"
+            
+    ioda:
+      backend: netcdf 
+      obsdataout: "/scratch1/NCEPDEV/da/Praveen.Kumar/JEDI/IODA_CONVR_ADPUPA_PB_2020110112/testoutput/gdas.t12z.adpupa3.prepbufr.nc"
+
+      dimensions:
+        - name: CloudSequence
+          path: "*/CLOUDSEQ"
+        - name: PressureEvent
+          path: "*/PRSLEVEL/P___INFO/P__EVENT"
+        - name: HumidityEvent
+          path: "*/PRSLEVEL/Q___INFO/Q__EVENT"
+        - name: TemperatureEvent
+          path: "*/PRSLEVEL/T___INFO/T__EVENT"
+        - name: HeightEvent
+          path: "*/PRSLEVEL/Z___INFO/Z__EVENT"
+        - name: WindEvent
+          path: "*/PRSLEVEL/W___INFO/W__EVENT"
+        - name: SeaTemperatureEvent
+          path: "*/SST_INFO/SSTEVENT"
+
+      variables:
+
+        # MetaData
+        - name: "MetaData/stationIdentification"
+          coordinates: "longitude latitude"
+          source: variables/stationIdentification
+          longName: "Station ID"

```

Comment:
Station Identification. Just for keeping things consistent among all conversion yamls

Reply:
I will see if this needs to be updated for bufr_ncep_prepbufr_adpupa.yaml.

Reply:
Fixed.